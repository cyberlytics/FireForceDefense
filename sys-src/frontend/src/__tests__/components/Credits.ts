import { mount, RouterLinkStub } from '@vue/test-utils';
import Credits from '@components/credits.vue';

describe('Credits.vue', () => {
    test('credits match snapshot', () => {
        const wrapper = mount(Credits, {
            mocks: {
                $t: (key: string) => key,
            },
            stubs: {
                RouterLink: RouterLinkStub,
            },
        });
        expect(wrapper.element).toMatchSnapshot();
    });
});
