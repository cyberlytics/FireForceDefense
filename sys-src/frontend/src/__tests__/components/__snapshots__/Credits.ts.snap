// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Credits.vue credits match snapshot 1`] = `
<div
  class="main-menu-bg"
>
  <div
    class="main-menu"
  >
    <div>
      <img
        alt="FireForceDefense"
        class="img-fluid"
        src="/assets/logo/logo.png"
        style="width: 250px;"
      />
    </div>
     
    <h2>
      Credits
    </h2>
     
    <p>
      Cameron Barbee
    </p>
     
    <p>
      Christian Piffel
    </p>
     
    <p>
      Philipp Stangl
    </p>
     
    <p>
      Sebastian Schuscha
    </p>
     
    <p>
      Thomas Ammann
    </p>
     
    <p>
      Thomas Stangl
    </p>
     
    <p>
      Tim Hoffmann
    </p>
     
    <p>
      Tobias Schotter
    </p>
     
    <hr
      class="mb-3"
    />
     
    <p
      class="small"
    >
      
            Feuerball-Animation: CC BY 4.0 by "EK"
      <br />
       
      <a
        href="https://lottiefiles.com/13581-dancing-fire"
      >
        https://lottiefiles.com/13581-dancing-fire
      </a>
    </p>
     
    <hr
      class="mb-3"
    />
     
    <div>
      <a
        class="btn btn-primary btn-block btn-lg"
      >
        back
      </a>
    </div>
  </div>
</div>
`;
